### PR TITLE
Added GPS popup

### PR DESCRIPTION
**Pull request by funtastix**
I've added a GPS popup similar to the signal strength popup, so that I can see the number of satellites and HDOP when I received PreArm errors on HDOP. Others might find this useful as well.

### CODE REVIEW

File: Android/res/layout/popup_info_gps.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    >
+
+    <TextView
+        android:id="@+id/bar_gps_satno"
+        style="?android:attr/buttonBarButtonStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="left|center_vertical"
+        android:textColor="@android:color/black"
+        android:text="Satellites 3"
```
Comment (ne0fhyk) [inline Android/res/layout/popup_info_gps.xml]: @funtastix the default `android:text` value should be the empty content string: [`@string/empty_content`](https://github.com/DroidPlanner/droidplanner/blob/9605649e4269c363819d7c01e069d9061f7a0141/Android/res/values/strings.xml#L453)

File: Android/res/layout/popup_info_gps.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        android:textSize="@dimen/info_bar_text_size"/>
+
+    <TextView
+        android:id="@+id/bar_gps_hdop"
+        style="?android:attr/buttonBarButtonStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="left|center_vertical"
+        android:textColor="@android:color/black"
+        android:text="HDOP 9.0"
```
Comment (ne0fhyk) [inline Android/res/layout/popup_info_gps.xml]: same as above.

File: Android/src/org/droidplanner/android/fragments/actionbar/ActionBarTelemFragment.java [hunk 3]
```diff
@@ -311,11 +321,17 @@ else if (signalStrength >= 25)
     private void updateGpsTelem() {
         final Drone drone = getDrone();
 
+        final View popupView = gpsPopup.getContentView();
+        TextView satNoView = (TextView) popupView.findViewById(R.id.bar_gps_satno);
+        TextView hdopView = (TextView) popupView.findViewById(R.id.bar_gps_hdop);
```
Comment (ne0fhyk) [inline Android/src/org/droidplanner/android/fragments/actionbar/ActionBarTelemFragment.java]: @funtastix if the `gps hdop` is displayed by default, then the `hdopView` should have its visibility set to `gone`, avoiding a repeat of the same information.

**Comment by ne0fhyk** [comment]
@funtastix thanks for the PR! 
I have added some comments about a couple issues I've seen before it's ready to be merged in.

**Comment by funtastix** [comment]
I've made the following adjustments:
- used empty string in layout
- depending if HDOP is shown through settings, show either HDOP or fix type/status as second value

**Comment by ne0fhyk** [comment]
Great! merging...

**Pull request merged**